optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting/persistence/VehicleRoutingImporter.java

```diff
@@ -391,6 +391,8 @@ private void createVehicleList() {
                 vehicle.setId(id);
                 id++;
                 vehicle.setCapacity(capacity);
+                //Multiple Depot VRP assign one vehicle for each depot
+                depotList.get(i % depotList.size());
```

Comment:
This actually improves the solution for VRPs with multiple depots. Attached are the screenshots of the solution before and after the fix. I ran belgium-road-time-multi-depot-n40-k10.vrp. This file has 10 depots. 

![after](https://cloud.githubusercontent.com/assets/23272809/21992496/ab174d60-dbe4-11e6-8666-d40e142557f5.png)
![before](https://cloud.githubusercontent.com/assets/23272809/21992506/b2cc1ae0-dbe4-11e6-99ad-b3c714de5d09.png)